application/backend/src/main/java/pl/ds/bulma/components/models/IconComponent.java

```diff
@@ -52,4 +60,38 @@ public class IconComponent {
   @Getter
   private boolean colorOnText;
 
+  @Inject
+  @Getter
+  @Default(values = "materialdesign")
+  private String iconLibType;
+
+  @Getter
+  private String containerSize;
+
+  @Inject
+  @Getter
+  @Default(values = "mdi mdi-36px")
+  private String iconSize;
+
+  @SlingObject
+  private Resource resource;
+
+  @PostConstruct
+  private void init() {
+    IconContainerService iconContainerService = new IconContainerService(this.resource);
+    String mappingPath = "/libs/bulma/components/common/icon/containersize/defaultsizemappings";
```

Comment:
You can use a path starting from `bulma/` so it should be possible to be overwritten in a project.